blackbird_python/blackbird/error.py

```diff
@@ -102,7 +104,9 @@ def syntaxError(self, recognizer, offendingSymbol, line, column, msg, e):
             "?",
         }:
             # inform the user of invalid symbol usage
-            error_msg = "Blackbird SyntaxError (line {}:{}): {} is not a valid Blackbird symbol."
+            error_msg = (
+                "Blackbird SyntaxError (line {}:{}): {} is not a valid Blackbird symbol."
+            )
             raise BlackbirdSyntaxError(
                 error_msg.format(line, column + 1, offendingSymbol.text)
             ) from None
```

Comment:
Are these newlines due to the line length setting in the formatter, or is this how it likes to format raise statements anyway? I'd increase the line length limit to 100 or even 120 chars to avoid this.

Reply:
I believe that's `black` which enforces this, isn't it @josh146 ?

Reply:
Yep. Black has an option to set the line length (`black file.py -l 100`), so I may rerun it right before merging.